chore: manifesto catch-up (#154)
adobe/helix-home: manifesto.md

### No Meetings `#t`

> Virtual Get-togethers, Show-&-Tells and [hackathons](./hackathons/README.md) only.

Reviewer: ```suggestion
(Virtual) Get-togethers, Show-&-Tells and [hackathons](./hackathons/README.md) only.
```

See the brackets as an expression of my hope that we will get out of our basements before we hit significant revenue.

Results of commits can be seen in a production environment in 60 seconds or less.

> 

Reviewer: ```suggestion
### MVP Done Right

It's "minimum" when the team has doubts that it could work. It's viable when users don't care and use it anyway.
```

Maybe a more accurate description of our approach to MVP. We don't declare MVP because we have run out of time and need to ship, we ship something that isn't proven and iterate on the feedback.

Reviewer 2: A bit cryptic, but ok... 😄

Make maximum use of available services.

### Lighthouse Score `#a`

> Fast and accessible. With tools to help improve enduser performance.

Reviewer: ```suggestion
Fast and accessible. With tools to help improve end user performance.
```
